docs/basics/data-center-endpoints.md

```diff
@@ -137,7 +137,7 @@ Depending on the framework or driver you use, you might need to make additional
 <td>*.tunnels.eu-west-3-lnbf.saucelabs.com</td><td><code>34.141.28.96/32</code></td>
 </tr>
 <tr>
-<td>Other endpoints</td><td><code>185.94.24.0/22</code></td>
+<td>Other endpoints</td><td><code>34.107.82.96/27</code><br/><code>185.94.24.0/22</code></td>
```

Comment:
context 

> ondemand.eu-central-1.saucelabs.com has address 185.94.26.249

old 185.* IP block will be deprecated (EOL date??) and switch to new IP block as above